static/app/views/explore/components/suspectTags/charts.tsx

```diff
@@ -170,7 +198,10 @@ function Chart({
         ? {adjective: 'different', message: 'This is suspicious.'}
         : {adjective: 'similar', message: 'Nothing unusual here.'};
 
-      return `<div style="max-width: 200px; white-space: normal; word-wrap: break-word; line-height: 1.2;">${selectedParam?.name} <span style="color: ${theme.textColor};">is <strong>${status.adjective}</strong> ${isDifferent ? 'between' : 'across'} selected and baseline data. ${status.message}</span></div>`;
+      const name = selectedParam?.name ?? baselineParam?.name ?? '';
+      const truncatedName = name.length > 300 ? `${name.slice(0, 300)}...` : name;
+
+      return `<div style="max-width: 200px; white-space: normal; word-wrap: break-word; line-height: 1.2;">${truncatedName} <span style="color: ${theme.textColor};">is <strong>${status.adjective}</strong> ${isDifferent ? 'between' : 'across'} selected and baseline data. ${status.message}</span></div>`;
```

Comment:
nit, would be nice if this is nicely formatted. I think that's possible because you're using string literals